lib/sejfguru_web/templates/asset/index.html.eex

```diff
@@ -1,4 +1,10 @@
 <div class="container">
+  <%= form_for @conn, asset_path(@conn, :index), [method: :get, class: "navbar-form navbar-left search"], fn f-> %>
+    <div class="form-group">
+      <input type="text" name="search" class="form-control" placeholder="Search">
```

Comment:
why not use form helpers for generating text input and button? https://hexdocs.pm/phoenix_html/Phoenix.HTML.Form.html#module-with-connection-data